test/e2e/specs/editor/various/inserting-blocks.spec.js

```diff
@@ -386,6 +386,320 @@ test.describe( 'Inserting blocks (@firefox, @webkit)', () => {
 
 		await expect( blockLibrary ).toBeHidden();
 	} );
+
+	test( 'should insert block with the slash inserter when using multiple words', async ( {
+		admin,
+		editor,
+		page,
+	} ) => {
+		await admin.createNewPost();
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '/tag cloud' );
+
+		await expect(
+			page.getByRole( 'option', { name: 'Tag Cloud', selected: true } )
+		).toBeVisible();
+		await page.keyboard.press( 'Enter' );
+
+		await expect(
+			editor.canvas.getByRole( 'document', { name: 'Block: Tag Cloud' } )
+		).toBeVisible();
+	} );
+
+	// Check for regression of https://github.com/WordPress/gutenberg/issues/23263
+	test( 'inserts blocks at root level when using the root appender while selection is in an inner block', async ( {
```

Comment:
The test no longer checks for original regression. The root appender is no longer displayed when the child block is selected, hence the need to clear block selection.

Should I remove the test or change it to use an in-between inserter that is as close as possible to the original test?

Cc @talldan, since you worked on the original fix and the test.

Reply:
Sounds like it can be removed 👍